assets/app/styles/_components.less

```diff
@@ -266,7 +296,7 @@
 .pod-block {
   .overview-pods {
     margin-right: 10px;
-    min-width: 190px;
+    min-width: 200px;
```

Comment:
We can switch to margin, but that has the effect of pod-templates not aligning horizontally when the scaling controls are not present.  Is that what we want?

<img width="765" alt="screen shot 2016-03-23 at 9 41 00 am" src="https://cloud.githubusercontent.com/assets/895728/13987008/e7c7e73c-f0db-11e5-80a6-b88a06fc933d.PNG">